docs/sources/alerting/manage-notifications/template-notifications/create-edit-and-delete-message-templates.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases:
+keywords:
+  - grafana
+  - alerting
+  - notifications
+  - templates
+  - create templates
+  - edit templates
+  - delete templates
+title: Create, edit and delete message templates
+weight: 200
+---
+
+# Create, edit and delete message templates
+
+Message templates can be found in the Contact points tab on the Alerting page. Here you can see a list of all your message templates. A message template can contain more than one template. For example, you might want to create a message template called `email` that contains all email templates.
+
+## Create message templates
+
+Step 1: Click the New template button:
+
+{{< figure max-width="940px" src="/static/img/docs/alerting/unified/list-message-templates-9-3.png" caption="List message templates" >}}
+
+Step 2: Choose a name for the message template. This name is not the name of the template that you will use in notification policies, but the name of the template as it appears in the Contact points page. The templates that you will use in notification policies should be written in the content field. For example, the screenshot below shows a template called `email.subject`:
```

Comment:
This is confusing-we should chat about behaviour and look at a way to rephrase.

Reply:
The procedure for create a template ends after naming it - shouldn't it continue with "add the content in the Content field". For information on how to write content, refer to write-templates...?

Reply:
I think you've left more screenshots in than we discussed;)

Reply:
> This is confusing-we should chat about behaviour and look at a way to rephrase.

OK! Let's chat about it this week! 🙂  I think we need some sentence about this somewhere because it's a confusing feature of the Grafana.

> The procedure for create a template ends after naming it - shouldn't it continue with "add the content in the Content field". For information on how to write content, refer to write-templates...?

You can create an empty message template by saving it without content, but I don't think we should encourage it in the documentation. The issue here is that we might mislead the user into thinking that their template is finished when in fact its just an empty template that cannot be used.